docs/translating/hindi.md

```diff
@@ -26,7 +26,7 @@ Zulip friendly and usable.
 ## Terms(शर्तें)
 
 - Message - **संदेश**
-- Private message (PM) - **निजी संदेश**
+- Direct Message (DM) - **Direct Message (DM)**
```

Comment:
Looks like we don't have a translation for Direct Message or DM to Hindi in Transifex yet.

Reply:
The parts that are not translated don't need to be documented then .. ( I had this confusion so I just wrote the msg shown in zupin on translation)